wled00/data/settings_wifi.htm

```diff
@@ -52,13 +52,13 @@
 				}
 				scanLoops = 0;
 
-				let cs = gId("CS");
-				if (cs) {
+				let cs = d.querySelectorAll("#wifi_entries input[type=text]#CS0");
```

Comment:
The scan button should work on every network, not just on the first one

Reply:
Fixed.